### PR TITLE
Changed to use strconv.ParseBool

### LINKED ISSUE #35: Handling boolean form values
Currently, we consider valid booleans to be only  the strings `"true"` and `"false"`, unlike `strconv.ParseBool()` which correctly converts `"1", "t", "T", "true", "TRUE", "True", "0", "f", "F", "false", "FALSE", "False"` into `bool`. We should discuss the implications of this and decide whether we want to use `strconv.ParseBool()` in the `safehttp.Form` implementation.

### PR DESCRIPTION
**Pull request by mattiasgrenfeldt**
Fixes #35

### CODE REVIEW

File: safehttp/form.go [hunk 1]
```diff
@@ -93,15 +93,12 @@ func (f *Form) Bool(param string, defaultValue bool) bool {
...
 		return defaultValue
 	}
-	switch vals[0] {
-	case "true":
-		return true
-	case "false":
-		return false
-	default:
-		f.err = fmt.Errorf("values of form parameter %q not a boolean", param)
+	b, err := strconv.ParseBool(vals[0])
```
Comment (mihalimara22) [inline safehttp/form.go:96]: The other functions use `paramVal` here. Idk which is more readable, but I think it makes the code more uniform to have the same name in all the functions.

Comment (empijei) [reply safehttp/form.go:96]: let's always use the same way to access params.

**Comment by kele** [comment]
@mihalimara22 @empijei please re-review since you've requested changes. I'd like to start doing clean merges. :)

**Comment by mattiasgrenfeldt** [comment]
Rebased on top of master, ready to merge

**Pull request merged**